docs/download-upload-clone/high-level-design.md

```diff
@@ -0,0 +1,56 @@
+# High Level Design
+
+* exporting a project exports its underlying scenarios
+* as the client agreed, pre-defined resources do not need to be described 
+  with unique identifier shared across possible instances of MarxanCloud 
+  (Databases) - if any shared resource with imported ID is not available, 
+  import should be rejected
+* each of the resource above should consider if it is allowed to be used 
+  (license issues). In case it cannot be used during export, system should ..
+  take under consideration
+* existing data converted from shapefiles should be converted back to 
+  shapefile format (currently the platform doesn't keep the uploaded files, 
+  rather consumes them)
```

Comment:
we need to confirm that we aren't dropping any data while consuming that may not be relevant for use in the instance but may be useful on import - none I can think of, but may need some cross checking.